syfertext/language.py

```diff
@@ -22,6 +23,9 @@
 from typing import Set
 from typing import Dict
 
+from collections import defaultdict
+import numpy as np
```

Comment:
numpy is not used anywhere previously,I converted it to torch tensor everywhere

Reply:
Done